contents/founders/cracked-manifesto.md

```diff
@@ -0,0 +1,87 @@
+---
+date: 2024-04-17
+title: Why you need to hire cracked engineers
```

Comment:
```suggestion
title: Hiring cracked engineers
```

Reply:
I don't feel super strongly about this, but always feel the simpler the headline, the better it is – especially if the goal is "get on HN."